site/_data/docs/handbook/toc.yml

```diff
@@ -1,3 +1,4 @@
+- url: /docs/handbook/components
```

Comment:
1. What's the difference between `/docs/handbook/...` and `/style-guide/`? 
2. My instinct is that `handbook` should be at root level, since its content applies to the whole site, not just stuff under `/docs/`